dias/analyzers/thermaldata_analyzer.py

```diff
@@ -131,8 +145,10 @@ def run(self):
                 )[0][0]
                 prod_sel.append(pidx)
 
-            # Load data
-            data = result.as_loaded_data(prod_sel=np.array(prod_sel))
+            # Load data from first acquisition
+            reader = andata.CorrReader(first_acquisition)
```

Comment:
Can this not reuse the reader from above (`read`)?

Reply:
It can! Updated.